### PR TITLE
PP-5646: Extra information on system error logs

### PR DESCRIPTION
**Pull request by oswaldquek**
We've had some instances where the secure URL have been fetched more than once;
however we currently don't log the gateway account information which would be
useful to do some analysis on.

## WHAT
_A brief description of the pull request:_

## HOW 
_Steps to test or reproduce:_

### CODE REVIEW

File: app/controllers/secure_controller.js [hunk 1]
```diff
@@ -47,7 +50,7 @@ exports.new = async function (req, res) {
         'the frontend state cookie not existing, or the frontend state cookie containing an invalid value.')
       return responseRouter.response(req, res, 'UNAUTHORISED')
     }
-    logging.systemError('Secure controller token', correlationId, chargeTokenId)
+    logging.systemError('Secure controller token', correlationId, chargeId, gatewayAccountId, gatewayAccountType)
```
Comment (jankowiakmaria) [inline app/controllers/secure_controller.js]: Is `chargeTokenId` the same as `chargeId` or is the change from one to another intentional?
(given the `logSystemError` function it looks like it should have been `chargeId` from the start)

File: app/controllers/secure_controller.js [hunk 2]
```diff
@@ -20,11 +20,14 @@ const paths = require('../paths')
 exports.new = async function (req, res) {
   const chargeTokenId = req.params.chargeTokenId || req.body.chargeTokenId
   const correlationId = req.headers[CORRELATION_HEADER] || ''
+  var chargeId
+  var gatewayAccountId
+  var gatewayAccountType
```
Comment (jankowiakmaria) [inline app/controllers/secure_controller.js]: [minor] `var` -> `let`

File: app/controllers/secure_controller.js (hunk 1 above)
Comment (oswaldquek) [reply app/controllers/secure_controller.js]: `chargeTokenId` is different from `chargeId`. yup, it should have been `chargeId` from the start, which this PR fixes.

File: app/controllers/secure_controller.js (hunk 2 above)
Comment (oswaldquek) [reply app/controllers/secure_controller.js]: what's the advantage of `let` here?

File: app/controllers/secure_controller.js (hunk 1 above)
Comment (jankowiakmaria) [reply app/controllers/secure_controller.js]: That's what I though (but wanted to confirm). 👍

File: app/controllers/secure_controller.js (hunk 2 above)
Comment (jankowiakmaria) [reply app/controllers/secure_controller.js]: In this particular case it doesn't make functional difference.
`let` is always more appropriate to use in terms of scoping. (`var` should really be avoided).
It's a minor comment - it doesn't block this PR. I was concerned about the other thing (that's why I haven't approved previously).

**Pull request merged**